frontend-cert/js-projects/second-functions/script.js

```diff
@@ -0,0 +1,32 @@
+function askQuestion(question) {
+    return prompt(question);
+}
```

Comment:
I wonder if we should get rid of this - you could just put `prompt(question)` in the `forEach`? That would simplify things a little I think. The user stories would need to be adjusted slightly.